snitchpy/__init__.py

```diff
@@ -528,13 +553,17 @@ def shutdown(self, *args):
 
     def _heartbeat(self):
         async def call():
-            req = protos.HeartbeatRequest(
-                session_id=self.session_id,
-            )
+            try:
+                req = protos.HeartbeatRequest(
+                    session_id=self.session_id,
+                )
 
-            return await self.grpc_stub.heartbeat(
-                req, timeout=self.grpc_timeout, metadata=self._get_metadata()
-            )
+                return await self.grpc_stub.heartbeat(
+                    req, timeout=self.grpc_timeout, metadata=self._get_metadata()
+                )
+            except Exception:
+                # Lost connection. Retry will occur in register
```

Comment:
Same here - if there's a logger, we could log this in debug so we know what's going on when debugging.

Reply:
done!